base/Dockerfile

```diff
@@ -18,12 +18,12 @@ ADD https://github.com/AcalephStorage/kviator/releases/download/v${KVIATOR_VERSI
 # Download confd
 ADD https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 /usr/local/bin/confd
 
-# Install prerequisites
-RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime && \
+# install prerequisites
+RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime python-setuptools udev && \
```

Comment:
`ceph-disk` invokes `udev` and `udevadm` (e.g. [here](https://github.com/ceph/ceph/blob/master/src/ceph-disk/ceph_disk/main.py#L1399))